packages/bsp/common/etc/default/armbian-motd.dpkg-dist

```diff
@@ -7,7 +7,7 @@
 MOTD_DISABLE=""
 ONE_WIRE=""
 HIDE_IP_PATTERN="^dummy0|^lo"
-PRIMARY_INTERFACE="$(ip route | grep '^default' | sed "s/.*dev //" | cut -d" " -f1)
+PRIMARY_INTERFACE="$(ip route | grep '^default' | sed "s/.*dev //" | cut -d" " -f1)"
```

Comment:
If both interfaces should be printed for the default route, then I approve.

Reply:
How do you mean?

Reply:
If two interfaces are available, and one of them cannot be used to access the Internet, for example, the traffic limit has ended, we will need a more complex selection logic using the ping utility.

Reply:
> How do you mean?

`PRIMARY_INTERFACE=eth0` **or** `PRIMARY_INTERFACE="eth0 wlan0"`
```
echo "$(ip route | grep '^default' | sed "s/.*dev //" | cut -d" " -f1)"
eth0
wlan0
```